packages/devtools_app/test/debugger_screen_test.dart

```diff
@@ -26,9 +28,12 @@ void main() {
   DebuggerScreen screen;
   FakeServiceManager fakeServiceManager;
   MockDebuggerController debuggerController;
+  MockProgramExplorerController mockProgramExplorerController;
   fakeServiceManager = FakeServiceManager();
   when(fakeServiceManager.connectedApp.isProfileBuildNow).thenReturn(false);
+  mockProgramExplorerController = MockProgramExplorerController.withDefaults();
   setGlobal(ServiceConnectionManager, fakeServiceManager);
+  setGlobal(ProgramExplorerController, mockProgramExplorerController);
```

Comment:
we should initialize variables, call `setGlobal` and set up mocks in a `setUp` method, not directly in main. This will allow each test to start with the same state instead of potentially having left over effects from the previous test